### ISSUE TITLE
Cannot open file or folder

### ISSUE THREAD

**Issue by wff75** [LightTable/LightTable#2363]
After downloading and extracting Light Table on Win 10 (Version 10.0.16299) I was ready to try out this seemingly cool editor. But, I couldn't even open a javascript file...

 1. Open File -> Select xxx.js and get the following msg in the bottom of the window saying the file does not exist.

 2. Open Folder -> Select a folder with multiple javascript files and you see the folder in the workspace. Select it to see files under it, they show for a split second and then am shown message telling me how to add a new file.

In summary. A complete waste of time. Shame as the video on the main website was enticing. Now on to VS code as that works really well...
